data/com.solus-project.brisk-menu.gschema.xml

```diff
@@ -26,5 +26,10 @@
       <summary>Search bar position</summary>
       <description>Where the search bar will appear in the menu</description>
     </key>
+    <key type="s" name="hot-key">
+      <default>'&lt;Ctrl&gt;F10'</default>
```

Comment:
I ponder if this shouldn't be using CDATA notation, and if it shouldn't default to one of the Super keys  too?